i18n/en.json

```diff
@@ -2014,6 +2342,10 @@
     "id": "api.remote_cluster.service_not_enabled.app_error",
     "translation": "The remote cluster service is not enabled."
   },
+  {
+    "id": "api.remote_cluster.update.app_error",
+    "translation": ""
```

Comment:
Empty?

Reply:
I think it came from the feature branch. PR: #16768